api/docs/deployment.dox

```diff
@@ -1,5 +1,5 @@
 /* ******************************************************************************
```

Comment:
In the section on "Cross-Compiling for ARM on Linux" in building.dox, the cmake command could also be updated to use the arm64 config.

Reply:
Added a new section above it for cross-compiling for 64-bit arm.